### ISSUE TITLE
Enhance slc loopback:model generator to display Warning when no datasource available

### ISSUE THREAD

**Issue by gunjpan** [strongloop/generator-loopback#176]
Current behavior of `slc loopback:model` generator:
After creating an empty LoopBack project, while creating a new model using `slc loopback:model` or `apic create --type model`, the generator doesn't display following prompt 
```console  
? Select the data-source to attach newModel to: (Use arrow keys)` 
```
until a datasource is defined. Also, it doesn't warn users about it.

This should be enhanced to show a warning message for `ease-of-use`.

**Comment by richardpringle** [comment]
Please add a documentation issue to go along with this and assign it to Rand

**Comment by 0candy** [comment]
For doc:
Add a note stating that if you select an empty app it does not come with any ds, not even the in memory one.
So if you go on to run the application, you would not see any of your models as there was no datasource attached.

**Comment by bajtos** [comment]
Published in `generator-loopback@1.22.0`.
